.github/workflows/build.yml

```diff
@@ -179,7 +179,7 @@ jobs:
       matrix:
         # most recent LTS releases as well as latest stable builds
         # https://github.com/ClickHouse/ClickHouse/pulls?q=is%3Aopen+is%3Apr+label%3Arelease
-        clickhouse: ["23.8", "24.3", "24.6", "latest"]
+        clickhouse: ["22.8", "23.3", "23.7", "latest"]
```

Comment:
Please use existing one ` ["23.8", "24.3", "24.6", "latest"]`
I've updated it recently and your change is going to revert it =)